### ISSUE TITLE
Zabbix 5.0 Support

### ISSUE THREAD

**Issue by himpierre** [zbx-sadman/BigBlueButton#6]
Hi sadman.

Thanks for your work. Would be great if you could port the template to Zabbix 5.0.

cheers, t.

**Comment by dmfox00** [comment]
Григорий, добрый день!
Подскажите, пожалуйста, будут ли конвертированы шаблоны до новых версий Заббикса?
Спасибо.

**Comment by zbx-sadman** [comment]
У меня сейчас эксплуатируется на Zabbix 6.0.14
![image](https://github.com/zbx-sadman/BigBlueButton/assets/12827470/178501e7-a304-4708-905b-49b3270b9d19)

А в чем затруднение?

**Comment by dmfox00** [comment]
Как понимаю, он не поддерживает новую версию. Или я не правильно понимаю?

![Screenshot_19](https://github.com/zbx-sadman/BigBlueButton/assets/151909737/73260717-35b3-4228-89f2-fb738393bbdc)

**Comment by zbx-sadman** [comment]
Я выгрузил в [Templates ](https://github.com/zbx-sadman/BigBlueButton/tree/main/Zabbix_Templates) шаблон со своего Zabbix 6, т.к. не особо помню, что там менял (если вообще менял).

Но, по скриншоту, мне кажется, что проблема на стороне агента - может не перезагрузили после правки конфига.

**Comment by dmfox00** [comment]
> Я выгрузил в [Templates ](https://github.com/zbx-sadman/BigBlueButton/tree/main/Zabbix_Templates) шаблон со своего Zabbix 6, т.к. не особо помню, что там менял (если вообще менял).
> 
> Но, по скриншоту, мне кажется, что проблема на стороне агента - может не перезагрузили после правки конфига.

Отлично! Этот шаблон работает. Спасибо и хорошего дня! =)

**Comment by dmfox00** [comment]
Добрый день! Подскажите, пожалуйста, Вы могли бы добавить в шаблон мониторинг участников с веб-камерами?

**Comment by zbx-sadman** [comment]
> Добрый день! Подскажите, пожалуйста, Вы могли бы добавить в шаблон
> мониторинг участников с веб-камерами?
>
 <name>[Total] Broadcasters</name>
<key>bbb.videosCount[allMeetings]</key>

**Comment by linuxmangr** [comment]
Григорий, добрый день!
Подскажите, пожалуйста, будут ли конвертированы шаблоны до новых версий до новых версий BBB
У меня версия  BigBlueButton Server 2.6.0 (1022)

**Comment by zbx-sadman** [comment]
Да, я планирую летом поднять версию своего BBB после физического апгрейда сервера.  Сразу прыгну на v2.7, полагаю.

Но, по пути могу и v2.6 поставить, посмотреть как заведется в мониторинге. В продакшне гонять, конечно не буду.

**Comment by linuxmangr** [comment]
> Да, я планирую летом поднять версию своего BBB после физического апгрейда сервера. Сразу прыгну на v2.7, полагаю.
> 
> Но, по пути могу и v2.6 поставить, посмотреть как заведется в мониторинге. В продакшне гонять, конечно не буду.

Было бы хорошо протестировать и на BigBlueButton Server 2.6.0 (1022) 
Так как я ставил 2.7 но отказался от его реализации так как выключили поддержку LDAP и ставит keykloac в Docker  что не совсем то что нужно.
Буду благодарен если протестируете.

**Comment by zbx-sadman** [comment]
> Было бы хорошо протестировать и на BigBlueButton Server 2.6.0 (1022)

Кажется, что дело было не в версиях BBB, а в просочившейся в шаблон опечатке.
Я заменил в репозитарии неверный, на тот, что в моей инсталляции вполне успешно работает с BBB v2.6

https://raw.githubusercontent.com/zbx-sadman/BigBlueButton/main/Zabbix_Templates/TemplateAppBigBlueButtonServer_Zabbix_v6.xml
